src/topology/local_homeomorph.lean

```diff
@@ -411,52 +430,140 @@ def prod (e : local_homeomorph α β) (e' : local_homeomorph γ δ) : local_home
 
 end prod
 
-section continuous_on
+section continuity
+
+/-- Continuity within a set at a point can be read under right composition with a local
+homeomorphism, if the point is in its target -/
+lemma continuous_within_at_iff_continuous_within_at_comp_right
+  {f : β → γ} {s : set β} {x : β} (h : x ∈ e.target) :
+  continuous_within_at f s x ↔
+  continuous_within_at (f ∘ e.to_fun) (e.to_fun ⁻¹' s) (e.inv_fun x) :=
+begin
+  split,
+  { assume f_cont,
+    have : e.to_fun (e.inv_fun x) = x := e.right_inv h,
+    rw ← this at f_cont,
+    have : e.source ∈ nhds (e.inv_fun x) := mem_nhds_sets e.open_source (e.map_target h),
+    rw [← continuous_within_at_inter this, inter_comm],
+    apply continuous_within_at.comp f_cont
```

Comment:
Again this could be `exact`. I'm not sure `exact` is really faster, but it may help maintenance to know that everything is meant to be there.

Reply:
Agreed. Replaced with `exact`.